### ISSUE TITLE
Anonymous editing

### ISSUE THREAD

**Issue by Kreijstal** [typst/typst#989]
### Description

Users who want to test new features, should also get a preview, of the editor, the signup wall, slows down adoption, we're not chatGPT 😂😂

### Use Case

For showing typst.app to a colleague on his computer really quick

**Comment by zouharvi** [comment]
Isn't that already possible by creating the share link (upper right in the editor)?

![image](https://user-images.githubusercontent.com/7661193/234499707-89b409b8-eb71-437f-aa57-8fbe781320f6.png)

**Comment by PgBiel** [comment]
> Isn't that already possible by creating the share link (upper right in the editor)?

I think they mean having something like `typst.app/playground` which would let you quickly compile some Typst in the browser without needing an account.

**Comment by Kreijstal** [comment]
> > Isn't that already possible by creating the share link (upper right in the editor)?
> 
> I think they mean having something like `typst.app/playground` which would let you quickly compile some Typst in the browser without needing an account.

Yes, I don't think this can be abused in a way, can it? Nobody needs to create an account to see if typst is the right choice for them, it's bad user experience.
